src/layouts/Home.vue

```diff
@@ -67,9 +66,8 @@
 
       <section id="enterprise-grade">
         <div class="section section-narrow no-bottom">
-          <h1 class="center">Built for Enterprise</h1>
-          <h3 class="center">Your design system’s missing half</h3>
-          <p class="center">Diez naturally integrates with any UI component solution in the world to help you create a <strong>complete design system.</strong></p>
+          <h1 class="center">Built for Design Systems</h1>
+          <p class="center">Diez is the perfect base-layer for your design system, supporting any UI component library or native codebase with a centralized set of visual styles.  Update once, update everywhere.</p>
```

Comment:
Not a typo but "Update once, update everywhere" seems a bit repetitive. Maybe "Update once, consume everywhere" or something like that?